### PR TITLE
Quantity

### PR DESCRIPTION
**Pull request by EllieLilithFox**
add qty field to gear and change frontend to match

### CODE REVIEW

File: gearcatalog/frontend/src/components/gear/EditGearForm.tsx [hunk 1]
```diff
@@ -25,6 +25,7 @@ match: { params } }: EditGearProps) => {
   const history = useHistory();
   const [gearUpdated, setGearUpdated] = useState(false);
   const [name, setName] = useState(selectedGear.name);
+  const [qty, setQty] = useState(0);
```
Comment (NatalieWolfe) [inline gearcatalog/frontend/src/components/gear/EditGearForm.tsx:28]: Why abbreviate the field name?

File: gearcatalog/frontend/src/components/gear/GearList.tsx [hunk 2]
```diff
@@ -51,7 +51,7 @@ const GearList = ({ deleteGear, getGear, gearList, token}: GearListProps) => {
                 <strong>{gear.name}</strong>
               </div>
               <div className="card-body">
-                <p className="card-text">Brand: {gear.brand} </p>
+                <p className="card-text">Quantity: {gear.qty}</p>
```
Comment (NatalieWolfe) [inline gearcatalog/frontend/src/components/gear/GearList.tsx:54]: What is the reason to replace brand with quantity?

Comment (EllieLilithFox) [reply gearcatalog/frontend/src/components/gear/GearList.tsx:54]: quantity seems more useful piece of info. When I had both it inserted a new line and it looked bad. So I went with qty.

File: gearcatalog/frontend/src/components/gear/EditGearForm.tsx (hunk 1 above)
Comment (EllieLilithFox) [reply gearcatalog/frontend/src/components/gear/EditGearForm.tsx:28]: Because I'm lazy and qty works.

Comment (NatalieWolfe) [reply gearcatalog/frontend/src/components/gear/EditGearForm.tsx:28]: Laziness is definitely the right thing to base naming on. ;)

**Review by NatalieWolfe — APPROVED** [review]

**Pull request merged**